ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -49,6 +49,7 @@
     - role: local_srcinstall
       src_tarball: https://github.com/protocolbuffers/protobuf/releases/download/v3.5.1/protobuf-cpp-3.5.1.tar.gz
       installed_target: /usr/local/bin/protoc
+      link_shared_libraries: True
```

Comment:
This role should likely be removed since this functionality isn't being used within Adopt builds atm.

Reply:
removed